apps/core/scenes/search/Search.js

```diff
@@ -294,14 +303,14 @@ class Search extends React.Component<Props, State> {
               onPress={this.handleDepartureDatePress}
               label="Departure"
               icon={<Icon name="calendar" />}
-              value={dateFrom}
+              value={DateFNS.format(dateFrom, submitDateFormat)}
             />
             {tripType === 'return' && (
               <TripInput
                 onPress={this.handleReturnDatePress}
                 label="Return"
                 icon={<Icon name="calendar" />}
-                value={returnFrom}
+                value={DateFNS.format(returnDateFrom, submitDateFormat)}
```

Comment:
what about DRY and make helper function for that something like `getFormatedDate(date)`?

Reply:
Ok, extracted.